jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererUtil.kt

```diff
@@ -126,7 +124,9 @@ object CodeWhispererUtil {
 
     private fun notifyConnectionExpired(project: Project, connection: BearerSsoConnection?) {
```

Comment:
The name can be more descriptive as `notifyConnectionExpiredRequestRelogin` to indicate the user needs to go through the login flow again.

Reply:
updated